Builds/CMake/RippledNIH.cmake

```diff
@@ -18,11 +18,14 @@ if (NOT DEFINED NIH_CACHE_ROOT)
 endif ()
 set (nih_cache_path
   "${NIH_CACHE_ROOT}/${gen_for_path}/${CMAKE_CXX_COMPILER_ID}_${CMAKE_CXX_COMPILER_VERSION}")
+set (nih_src_path "${NIH_CACHE_ROOT}/src")
```

Comment:
Can we bikeshed for a little on CMake sytle? I'd like to pick one style and commit to keeping at least new changes in that style. The primary question is: space or no space between directive and open parenthesis? I vote for no space, which is what the official CMake documentation uses (in most places, and seemingly in all new documentation; it is an inconsistent mish-mash after years of contributions though).

Reply:
It's probably fair to say our CMake configs are also a mish-mash after years of contributions, too. :) I have no problem with dropping that space.